vespalib/src/tests/invokeservice/invokeservice_test.cpp

```diff
@@ -29,6 +29,25 @@ TEST("require that wakeup is called") {
     EXPECT_EQUAL(countAtStop, a._count);
 }
 
+TEST("require that now is moving forward") {
+    InvokeCounter a;
+    InvokeServiceImpl service(1ms);
+    EXPECT_EQUAL(0u, a._count);
+    steady_time prev = steady_clock::now();
```

Comment:
I would prefer moving this above the creation of the invoke service to ensure now > prev also the first time

Reply:
Fixed